packages/extension-ui/src/Popup/Accounts/index.tsx

```diff
@@ -4,27 +4,17 @@
 
 import React, { useContext } from 'react';
 import styled from 'styled-components';
-import { AccountJson } from '@polkadot/extension-base/background/types';
 
 import QrImage from '../../assets/qr.svg';
 import AccountsTree from './AccountsTree';
 import { AccountContext, Button, Header, MediaContext, AddAccount, ButtonArea, Svg, ButtonWithSubtitle } from '../../components';
 
 type Props = {};
 
-function findMasterAccount (accounts: AccountJson[]): AccountJson | undefined {
-  return accounts
-    .map((account) => ({ ...account, whenCreated: account.whenCreated || Infinity }))
-    .sort((a, b) => a.whenCreated - b.whenCreated)
-    .find((account) => !account.isExternal);
```

Comment:
Shouldn't we move filtering out external acocunts to `buildHierarchy` now?

Reply:
That's why those lines are removed :)

Reply:
But in `buildHierarchy` there's no filter for external.

Reply:
Yep, you are right, missed that